docs/development/pigs/pig-011.rst

```diff
@@ -0,0 +1,274 @@
+.. include:: ../../references.txt
+
+.. _pig-011:
+
+*********************
+PIG 11 - Light curves
+*********************
+
+* Author: Régis Terrier, Christoph Deil, Axel Donath, David Fidalgo, Atreyee Sinha
+* Created: July 2, 2018
+* Accepted:
+* Status: draft
+* Discussion: `GH 1451`_
+
+Abstract
+========
+
+In this PIG we want to discuss a restructuring of the way light curves are
+computed and stored in Gammapy. Lightcurves in gamma-ray astronomy are the
+result of a series of fits of the source flux in each time bin. Lightcurve
+extraction covers therefore both the data reduction and the data modeling steps.
+The lightcurve estimation will therefore have these two steps.
+
+Here, we propose to perform the data reduction step in each of the time bins
+and store the result in the form of a ``Datasets`` (``MapDataset`` or
+``SpectrumOnOffDataset`` depending on the selected approach).
+Each individual ``Dataset`` is then modeled and fitted to extract the
+source flux and its errors in each time bin. The result is then stored in a
+``LightCurve`` object which contains a ``Table`` of the fit results.
+
+For this purpose, we propose to introduce two new classes to perform the data
+reduction first and then then the fitting. The time dependent data reduction
+could be a specific case of the high level pipeline, when a list of time bins
+is passed with the configuration ``dict``. The data fitting should be performed
+by the new :class:``LightCurveEstimator`` class, which  should essentially be a
+wrapper around the ``gammapy.spectrum.FluxPointEstimator`` class that does the same
+thing for spectrum and map datasets.
+
+Introduction
+============
+
+Lightcurves in gamma-ray astronomy
+----------------------------------
+
+In photon counting experiments, lightcurves are often simply obtained by counting
+events in a given energy range in a set of time bins. In ground based gamma-ray
+astronomy, things are usually more complex.
+
+The response and the instrumental background of the instruments can strongly vary
+over time on a night scale, e.g. because the source elevation changes or on longer time
+scales given the possible changes of the atmosphere transparency or the instrument
+efficiency.
+
+Another complexity comes from the astrophysical background which can often pollute a source
+and needs to be properly removed to extract the intrinsic source flux at any given time.
+
+For these reasons, gamma-ray lightcurve are usually the results of a fit of model on
+the data performed on a number of time bins to extract the source flux in these time
+bins.
+
+This is more limited than e.g. time resolved spectral analysis. Although the latter share
+many similarities with the lightcurve extraction, it is a more complex task which
+we do not cover here.
```

Comment:
OK, to not cover it in this PIG, but maybe mention in an "outlook" section how it could be achieved? E.g. by allowing more free parameters for the spectral model, or e.g. run the `FluxPointsEstimator`  per time bin.